pages/wg/dei.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Diversity, Equity, & Inclusion Working Group
+subtitle:
+permalink: /wg/dei/
+---
+
+# Overview
+
+The US-RSE Association is committed to providing an inclusive environment with equitable treatment for all and to promoting and encouraging diversity throughout the RSE community in the US. The US-RSE Diversity, Equity, and Inclusion (DEI) working group is for those who are interested in regular discussions regarding DEI in the RSE community. The goal of the DEI working group is to promote the values of the US-RSE DEI Mission Statement with a particular focus on promoting opportunities for DEI education to the wider community. The group meets bi-weekly via Zoom and welcomes all newcomers.
+
+To get involved with the DEI working group, visit the `#dei-discussion` channel on the US-RSE slack, or contact the <a href="mailto:dei_wg@us-rse.org">DEI working group list</a>.
+
+
+# Work Products
+
+The US-RSE DEI working group has produced and hosts the following documents and events.
+
+## DEI Mission Statement
+
+The US-RSE DEI working group's first large initiative was to create a US-RSE DEI Mission Statement to integrate into the US-RSE core values. This DEI Mission Statement was published in August 2021 on the [DEI Mission Statement page]({{ site.baseurl }}/about/dei-mission). We expect this mission statement to evolve and mature as the US-RSE community grows and welcome feedback.
```

Comment:
I just got this referenced in a call for a DEI plan at the lab - super proud of y'all!

Reply:
!!! Really?! That is so neat!